src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -538,26 +539,31 @@ bool ZoneSet::CalculateColumnsAndRowsLayout(Rect workArea, FancyZonesDataTypes::
 
     // Note: The expressions below are NOT equal to total{Width|Height} / zoneCount and are done
     // like this to make the sum of all zones' sizes exactly total{Width|Height}.
-    for (int zone = 0; zone < zoneCount; zone++)
+    for (int cnt = 0; cnt < zoneCount; ++cnt)
```

Comment:
Let's use meaningful names for variables: `cnt` -> `count` or even better `index` since you start from 0 and `count` in this case would not make sense.

Reply:
Done.